samples/README.md

```diff
@@ -0,0 +1,9 @@
+# Sample Details
+
+## Images
+
+All test images are from [Pexels](https://www.pexels.com) and licensed under a [CC0 License](https://creativecommons.org/publicdomain/zero/1.0/).
```

Comment:
Under CC0 perhaps? There is only one CC0 license per my understanding. People say "a CC license" because CC consists of 7 variants.